pandas/tests/series/test_analytics.py

```diff
@@ -32,36 +32,64 @@ class TestSeriesAnalytics(TestData):
     @pytest.mark.parametrize("use_bottleneck", [True, False])
     @pytest.mark.parametrize("method", ["sum", "prod"])
     def test_empty(self, method, use_bottleneck):
+        if method == "sum":
+            unit = 0.0
+        else:
+            unit = 1.0
 
         with pd.option_context("use_bottleneck", use_bottleneck):
             # GH 9422
             # treat all missing as NaN
             s = Series([])
             result = getattr(s, method)()
             assert isna(result)
+            result = getattr(s, method)(min_count=0)
```

Comment:
I would make explcity passing ``min_count=1`` in tests as well, and then *add* another entry w/o passing min_count (e.g. the default)